tests/time_tests/scripts/run_timetest.py

```diff
@@ -105,7 +105,7 @@ def run_timetest(args: dict, log=None):
     aggregated_stats = aggregate_stats(filtered_stats)
     log.debug("Aggregated statistics after full run: {}".format(aggregated_stats))
 
-    return 0, aggregated_stats
+    return 0, aggregated_stats, filtered_stats
```

Comment:
@asomsiko do we really want to save already filtered data, or it should be raw? I think we should save raw data, and, if required, apply any filtering manually